plugins/woocommerce/src/Internal/Admin/Settings.php

```diff
@@ -200,6 +200,12 @@ public function add_component_settings( $settings ) {
 			'installedPlugins' => PluginsHelper::get_installed_plugin_slugs(),
 			'activePlugins'    => Plugins::get_active_plugins(),
 		);
+
+		// DO NOT use outside of core, these can be removed without deprecation.
+		$settings['__experimentalFlags'] = array(
+			'isBranding2_0' => BrandingController::use_new_branding(),
+		);
```

Comment:
Hi, @Konamiman! Let me know if you have any thoughts of having this parameter embedded into `wcSettings.admin` in admin screens here.

Reply:
If the parameter needs to be used in fronted code then yes, it makes sense to make it available like that.

One nitpick, though: the `BrandingController::use_new_branding` method is temporary, it will be removed once the branding is rolled out for verybody (and thus the changes in this and other related PRs will have to be "undone", that's planned). So we can use a simpler name like `useNewBranding` instead of `isBranding2_0`.

Reply:
Thanks for the feedback!

> So we can use a simpler name like useNewBranding instead of isBranding2_0.

Good point, I changed all the "2.0" into newBranding in e3de242e344a0d4fd17a0043358bd226e4b0810c